managed/services/telemetry/telemetry_test.go

```diff
@@ -264,7 +263,7 @@ func initMockTelemetrySender(t *testing.T, expectedReport *reporter.ReportReques
 		var mockTelemetrySender mockSender
 		mockTelemetrySender.Test(t)
 		mockTelemetrySender.On("SendTelemetry",
-			mock.AnythingOfType(reflect.TypeOf(context.TODO()).Name()),
+			mock.Anything,
```

Comment:
That was the reason: https://github.com/vektra/mockery/issues/324

Mockery can properly work with contexts.